src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1099,4 +1099,26 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
   <data name="SourcesCommandValidProtocolVersion" xml:space="preserve">
     <value>The protocol version specified is invalid. Valid protocol versions are from {0} to {1}.</value>
   </data>
+  <data name="Error_HttpSources_Multiple" xml:space="preserve">
+    <value>You are running the '{0}' operation with 'HTTP' sources: {1}. NuGet requires HTTPS sources. To use HTTP sources, you must explicitly set 'allowInsecureConnections' to true in your NuGet.Config file. Please refer to https://aka.ms/nuget-https-everywhere for more information.</value>
+    <comment>0 - The command name. Ex. Push/Delete. 1 - list of server URIs</comment>
```

Comment:
The code seems to use source.Name, not the URI.
(There are other similar resx would need this update)

```suggestion
    <comment>0 - The command name. Ex. Push/Delete. 1 - list of source names</comment>
```

Reply:
Thanks for the review. Since I am working on a follow up PR addressing non-blocking issues with this PR, I will address your comments there.